core/src/main/java/org/p2p/core/token/TokenExtensions.kt

```diff
@@ -16,7 +15,7 @@ data class TokenExtensions(
     val isTokenCellVisibleOnWalletScreen: Boolean? = null,
     val tokenPercentDifferenceOnWalletScreen: Int? = null,
     val isCalculateWithTotalBalance: Boolean? = null,
```

Comment:
there no default values for Booleans? 
so if it's null - it's false by default?

Reply:
yeah if it's null it will be false by default

Reply:
so maybe we can remove null's and replace them with default values?